cowbird/config.py

```diff
@@ -191,7 +191,7 @@ def validate_sync_perm_config_schema(sync_cfg: SyncPointConfig) -> None:
             "services": {
                 str: {  # Service type, must correspond to an actual Magpie service type
                     str: [  # Resource key, used to identify the resource here and in the permissions_mapping
-                        {"name": str, "type": str, Optional("field"): str, Optional("regex"): str}
+                        {Or("name", "regex", only_one=True): str, "type": str, Optional("field"): str}
```

Comment:
I think it might be worthwhile to update the doc `configuration.rst` according to new changes to the config, explaining the new parameters added, their behaviour and use cases.

Reply:
No doc update applied. Is this still relevant? Maybe pinpoint which sections could need minor fixes. 
I'm fine with the examples in `config/config.example.yml` at the very least.

Reply:
I'm updating it